packages/helm/build.sh

```diff
@@ -0,0 +1,23 @@
+TERMUX_PKG_HOMEPAGE=https://helm.sh
+TERMUX_PKG_DESCRIPTION="Helm helps you manage Kubernetes applications" 
+TERMUX_PKG_LICENSE="Apache-2.0" 
+TERMUX_PKG_MAINTAINER="https://github.com/helm/helm/blob/master/OWNERS" 
```

Comment:
Maintainer filed should be either absent or be filled with username/email if package submitter. Specifying side maintainers (unrelated to @termux) and irrelevant data like file links is not allowed here.